cmd/batch.go

```diff
@@ -162,19 +163,25 @@ Changes are sorted in the following order:
 		false,
 		"Force a new version file even if one already exists",
 	)
+	cmd.Flags().StringVarP(
+		&b.Project,
+		"project", "j",
+		"",
+		"SPecify which project version we are batching",
```

Comment:
```suggestion
		"Specify which project version we are batching",
```

<img src="https://media0.giphy.com/media/WWaLzNhzxs6VllPc91/giphy.gif"/>